.gometalinter.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Vendor": true,
+  "Deadline": "2m",
+  "Sort": [
+    "linter",
+    "severity",
+    "path",
+    "line"
+  ],
+  "Exclude": [
```

Comment:
Is there a way to suppress an error in the code just before where it happens. This list has no context associated with it. I would like to know where `should have signature WriteByte\\(byte\\) error \\(vet\\)` happens. I may want to fix it some day. `.*DataOffset should have comment.*` like suppressions may shadow unintended suppression of new code.